politeiad/config.go

```diff
@@ -286,7 +289,7 @@ func loadConfig() (*config, []string, error) {
 		LogDir:     defaultLogDir,
 		HTTPSKey:   defaultHTTPSKeyFile,
 		HTTPSCert:  defaultHTTPSCertFile,
-		Version:    version.String(),
```

Comment:
This version wasn't being used in any part of the code, so it was removed.

Reply:
Leave this in.

Reply:
Leave this in.